en/releases/centreon-core.md

```diff
@@ -18,6 +18,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web release notes
 
+### 20.10.2
+
+#### Enhancements
+
+- [API] Add normalizers for data found in concordanceArray
+- [API] Get topology of servers of a Centreon Platform
+- [Configuration] Add a special variable for trap OID
+- [Configuration] Add pool size parameter in configuration for Centreon Broker
+- [Resources Status] Add alias & fqdn in host detail panels
+- [Resources Status] Add URL link button from host and service extended information configuration
+
+#### Bug fixes
+
+- [Authentication] New LDAP configurations are broken
+- [CLAPI] Export does not export default contactgroup linked to a LDAP configuration
+- [Configuration] PHP Warning while creating a Centreon Engine configuration
+- [Configuration] Unable to save log level in Centreon Engine form
+- [Knowledge Base] Access to mediawiki is very slow
+- [Resources Status] Display issue when resource has a configured icon
```

Comment:
Missing mention for ticket MON-5812 

[Frontend][Backend][events view] Incorrect default downtime duration
--